### PR TITLE
Add anatomy of a question help modal

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
- Adds a link which opens "The Anatomy of a Question" modal 

### How to review 
- Ensure tests pass
- Ensure modal opens with correct image

### CODE REVIEW

**Comment by adammahmood** [comment]
[Create link to open "The Anatomy of a question" (2)](https://trello.com/c/n0h2yoV3/740-create-link-to-open-the-anatomy-of-a-question-2)

File: eq-author/src/App/questionPage/Design/QuestionProperties/index.test.js [hunk 1]
```diff
@@ -39,4 +39,14 @@ describe("QuestionProperties", () => {
...
   });
+
+  it("should correctly handle HelpButton onClick", () => {
+    wrapper.find(HelpButton).simulate("click");
+    expect(wrapper.find(HelpModal).prop("isOpen")).toBeTruthy();
+  });
+
+  it("should correctly handle HelpModal onClose", () => {
+    wrapper.find(HelpModal).simulate("close");
+    expect(wrapper.find(HelpModal).prop("isOpen")).toBeFalsy();
```
Comment (tgandrews) [inline eq-author/src/App/questionPage/Design/QuestionProperties/index.test.js]: Would it have been falsy before the close action as well?

File: eq-author/src/App/questionPage/Design/QuestionProperties/index.js [hunk 2]
```diff
@@ -32,6 +35,21 @@ const PropertyDescription = styled.p`
   ${colors.darkGrey}
 `;
 
+export const HelpButton = styled.button`
+  --color-text: ${colors.primary};
+  margin: 0.5em 0;
+  position: relative;
+  left: -0.5em;
+  padding: 0;
+  font-size: 1em;
```
Comment (hamishtaplin) [inline eq-author/src/App/questionPage/Design/QuestionProperties/index.js]:
```suggestion
  font-size: 0.9em;
  text-align: left;
```

File: eq-author/src/App/questionPage/Design/QuestionProperties/index.test.js (hunk 1 above)
Comment (adammahmood) [reply eq-author/src/App/questionPage/Design/QuestionProperties/index.test.js]: I've merged this into the previous test which now tests the opening / closing of the modal together 👍

**Pull request merged**